bin/reth/src/node/mod.rs

```diff
@@ -444,24 +489,37 @@ impl Command {
         }
 
         let factory = reth_executor::Factory::new(Arc::new(self.chain.clone()));
+
+        let default_stages = if continuous {
+            StageSetBuilder::default()
+                .add_stage(HeaderStage::new(header_downloader, consensus.clone()).continuous())
+                .add_stage(TotalDifficultyStage::new(consensus.clone()))
+                .add_stage(BodyStage { downloader: body_downloader, consensus: consensus.clone() })
+                .add_set(OfflineStages::new(factory.clone()))
+                .add_stage(FinishStage::new(updater.clone()))
+        } else {
+            DefaultStages::new(
+                consensus.clone(),
+                header_downloader,
+                body_downloader,
+                updater.clone(),
+                factory.clone(),
+            )
+            .builder()
+        };
```

Comment:
created some helpers to make it less verbose, unfortunately once the headers stage is in the `StageSetBuilder`, it can't be modified because at that point it's a `Box<dyn Stage>`. Instead, the added helpers make building default stages with a custom headers stage easier